src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { colorMap, size, fontSize } from './theme';
+
+const Tab = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+`;
+
+const activeStyles = css`
+    color: ${colorMap.devWhite};
+    border-bottom-color: ${colorMap.darkGreen};
+`;
+
+const defaultStyles = css`
+    &:hover {
```

Comment:
for the `:hover`, we should be able to add it into a `styled(P)`, does that work?

Reply:
Hovering does work. However, because we are separating the text from the button when we do that, the text color doesn't seem to remain white (indicating it's active) once clicked.

Reply:
Hmm I tried adding a `P` and that seemed to do it? Here is what I had, does this work for you?

```
                        <TabButton
                            key={item}
                            isActive={isActive}
                            onClick={() => handleClick(item)}
                        >
                            <P collapse>{item}</P>
                        </TabButton>
```

Reply:
Actually should this `defaultStyle` hover be applied to the button all the time? (even if active?) That can reduce complexity here

Reply:
Sorry to clarify can we take this `hover` stuff and put it in `TabButton` directly since it is applied at all times? Then we can remove this `defaultStyles` var and remove this logic also from `activeStyles`